frontend/src/data/variables/BrfssProvider.test.ts

```diff
@@ -164,9 +170,39 @@ describe("BrfssProvider", () => {
     const rawAcsData = [AL_ACS_ASIAN_ROW, NC_ACS_ASIAN_ROW, NC_ACS_WHITE_ROW];
 
     // Create final rows with diabetes_count & diabetes_per_100k
-    const NC_ASIAN_FINAL = finalRow(NC, RACE, ASIAN_NH, 400, 40000, 16.7, 40);
-    const NC_WHITE_FINAL = finalRow(NC, RACE, WHITE_NH, 600, 60000, 83.3, 60);
-    const NC_ALL_FINAL = finalRow(NC, RACE, ALL, 1000, 50000, 100, 100);
+    const NC_ASIAN_FINAL = finalRow(
```

Comment:
Nit: I know it was like this before, and I also should do this for the other providers, but can you add comments on the params for the numbers for easier readability?